src/main/clojure/aerospike_clj/client.clj

```diff
@@ -58,84 +48,27 @@
   (let [elp (policy/map->event-policy conf)]
     (NioEventLoops. elp 1 true "NioEventLoops")))
```

Comment:
The number of event loops should be configurable, not hardcoded.
(The name too)